osu.Game/Utils/ModUtils.cs

```diff
@@ -121,7 +130,14 @@ public static bool CheckValidForGameplay(IEnumerable<Mod> mods, [NotNullWhen(fal
         /// <param name="invalidMods">Invalid mods, if any were found. Will be null if all mods were valid.</param>
         /// <returns>Whether the input mods were all valid. If false, <paramref name="invalidMods"/> will contain all invalid entries.</returns>
         public static bool CheckValidRequiredModsForMultiplayer(IEnumerable<Mod> mods, [NotNullWhen(false)] out List<Mod>? invalidMods)
-            => checkValid(mods, m => m.IsPlayable(ModUsage.MultiplayerGlobal), out invalidMods, true);
+        {
+            mods = mods.ToArray();
+
+            if (!CheckCompatibleSet(mods, out invalidMods))
```

Comment:
Not sure why this isn't handling `MultiMod`s while `CheckValidForGameplay()` is. The two should match in behaviour.

I can see two ways to go about it: either mirror the code in `CheckValidForGameplay()`, or split a separate `checkValid(mods, m => !(m is MultiMod), out invalidMods);` call preceding this in both places.

Reply:
Hmm, thought it's pointless to check that in `CheckValidForMultiplayerRequiredMods` since it shouldn't even be a valid in the game-wide mods bindable, but I guess that's irrelevant in this case as this could get sent directly to the server. Replacing with `!(m is MultiMod)` makes sense.